nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -340,6 +340,7 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
     "docker"
   ];
   ```
+- In `mastodon` it is now necessary to specify location of file with `PostgreSQL` database password. In `services.mastodon.database.passwordFile` parameter default value `/var/lib/mastodon/secrets/db-password` has been changed to `null`.
```

Comment:
We have missed the merge window for that release, the notes will have to be moved to the next release.

Reply:
Moved release notes.